tests/conftest.py

```diff
@@ -40,12 +40,9 @@ def _reset(self):
         self.account = None
         self.stack = []
 
-    def _sign_transaction(self, encoded_tx):
-        """Sign transaction data sent to the Ledger"""
-        tx = decode_transaction(encoded_tx)
-        resp = self.account.sign_transaction(tx.to_rpc_dict())
-
-        v = resp.v.to_bytes(1, "big")
+    def _resp_to_bytearray(self, resp):
+        # for large chain_id, v can be bigger than a byte. we'll take the lowest byte for the signature
+        v = resp.v.to_bytes(8, "big")[7].to_bytes(1, "big")
```

Comment:
suggestion(non-blocking): Since it's for the `MockDongle`, if tests pass I'll send it.   But a reference and/or unit test covering this would be super cool.

Reply:
the test `test_web3_middleware_legacy` does not pass without using 8 bytes for `v`, so it's somehow covered already. That test uses the chain id `131277322940537` from `pyevm`

Reply:
Curious.  The previous test run passed fine.  I didn't find anything from a brief pass of the changelog for eth_account.  

Could be a Python 3.11 change.  There was an update to `int.to_bytes()` but it just looks like the default args changed.

Can't re-run previous tests because of a breaking setuptools change, so just nee to push through with a full dep upgrade.